### ISSUE TITLE
Update configuration component: add constraints, add presets, reorder active/passive mode, add explainer

### ISSUE THREAD

**Issue by coderbyheart** [NordicSemiconductor/asset-tracker-cloud-app-aws-js#256]
## Add these constraints

If below [constraints](https://github.com/NordicSemiconductor/asset-tracker-cloud-docs/commit/efe0f39a5ea04ae36c820aa17aea94cd98a42f13) are not satisfied, user should not be able to submit configuration. Display helpful error messages.

- [x] `Movement resolution` > `Accelerometer inactivity timeout`
- [x] `Accelerometer activity threshold` > `Accelerometer inactivity threshold`

## Reorder sections

- [x] New order (also in mobile view):

1. Mode
2. GNSS Timeout
3. Active Mode Settings
4. Passive Mode Settings
5. Accelerometer Settings
6. Data Sample

## Add presets

Provide two presets, so the user can apply a configuration for a tracking scenarion with one click:

- [x] 1. Walking: "Select this preset if you want to track the movement of a person or a pet"
- [x] 2. Parcel: "Select this preset if you want to track the movement of a parcel."

Parcel config:

- passive mode
- movement resolution: 3600
- Accelerometer activity threshold: 10
- Accelerometer inactivity threshold: 5
- Accelerometer inactivity timeout: 1200
- Movement Timeout: 21600

Walking config:

- passive mode
- movement resolution: 300
- Accelerometer activity threshold: 10
- Accelerometer inactivity threshold: 5
- Accelerometer inactivity timeout: 60
- Movement Timeout: 3600

## Add config explainer

- [x] Add an explainer element, that explains in plain english what will happen, using the values from the config and also has clickable links to the respective input fields: "When in motion the tracker will send an update to the cloud every hour (movement resolution). When motion stops for more than 20 minutes (Accelerometer inactivity timeout), an update will be sent to the cloud. If not in motion an update will be sent to the cloud every 12 hours (Movement Timeout)."

**Comment by MLopezJ** [comment]
# New order of the fields was defined after considering the output of the initial requested order. 

## **New order:**
![thumbnail_Image](https://user-images.githubusercontent.com/8351865/191241302-00248c72-82b6-47f6-b626-48eaee90d8e4.jpg)


## **Output of the initial requested order:**  

### - Output 1 
![Screenshot from 2022-09-20 11-07-11](https://user-images.githubusercontent.com/8351865/191241213-8fc7155b-ca05-4e2c-9539-eef270b49af1.jpg)
 
### - Output 2
![Screenshot from 2022-09-20 11-04-48](https://user-images.githubusercontent.com/8351865/191241234-e3e47d01-02b4-4907-a6a3-92ef53b09ecd.jpg)
